common/jmeter/setup/TestData_Add_Tenants.jmx

```diff
@@ -23,11 +23,15 @@
             <stringProp name="Argument.value">${__P(port,9443)}</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-          <elementProp name="adminCredentials" elementType="Argument">
-            <stringProp name="Argument.name">adminCredentials</stringProp>
-            <stringProp name="Argument.value">YWRtaW46YWRtaW4=</stringProp>
+          <elementProp name="username" elementType="Argument">
+            <stringProp name="Argument.name">username</stringProp>
+            <stringProp name="Argument.value">admin</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="superAdminPassword" elementType="Argument">
+            <stringProp name="Argument.name">superAdminPassword</stringProp>
```

Comment:
Since above is username, let's make this just password.

Reply:
Addressed